<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolForm</class>
 <widget class="QWidget" name="ToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="toolListLayout">
       <item>
        <layout class="QHBoxLayout" name="toolsAddRemoveLayout">
         <item>
          <widget class="QLabel" name="toolsLabel">
           <property name="text">
            <string>Tool</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="toolsSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolsAddButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolsRemoveButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="toolsListView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="toolNumberLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Tool Number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="toolNumberSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="toolHolderLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Tool Holder</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="toolHolderComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="geometryLayout">
           <item>
            <widget class="QLabel" name="geometryLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Geometry</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="geometrySpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="geometryAddButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="geometryRemoveButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="geometryTable">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
